.travis.yml

```diff
@@ -67,6 +67,7 @@ addons:
 before_install:
   - jdk_switcher use oraclejdk8
 before_script:
+  - ccache -M 1G
```

Comment:
Why two ccache lines instead of 1?  Can you add a comment to explain what / why these options are being used?

Reply:
I suppose there is no real reason for having them in separate command invocation, other than the fact that they are logically separate actions. The `-M 1G` just sets the max cache size to 1GB (the default is 512MB, and just building the vm once puts the cache usage at 517MB)